Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -157,6 +158,13 @@ class BoolSetting: Setting {
         control.addTarget(self, action: "switchValueChanged:", forControlEvents: UIControlEvents.ValueChanged)
         control.on = prefs.boolForKey(prefKey) ?? defaultValue
         cell.accessoryView = control
+        cell.detailTextLabel?.snp_makeConstraints { make in
```

Comment:
Extracting the if so we don't needlessly call makeConstraints if we don't need to.